JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,20 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  const divisor = 60;
+
+  if (seconds <= 0) {
+    return console.log("The function only accepts positive numbers");
+  } else {
```

Comment:
Since you are returning a value you can skip the else block. In this way, you prevent nested code blocks.
```suggestion
  if (seconds <= 0) {
    return console.log("The function only accepts positive numbers");
  }
```